FirebaseCore/CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+# Unreleased
+- [changed] Firebase now requires at least Xcode 16.2. See
+  https://developer.apple.com/news/?id=9s0rgdy9 for more info.
+
```

Comment:
Hi @mikehardy, heads up that firebase-ios-sdk's 10.12 release will require Xcode 16.2. This is in preparation for Apple's April 24 deadline to start releasing apps with Xcode 16: https://developer.apple.com/news/?id=9s0rgdy9

Reply:
Ah - thanks for the heads up - we've got another similarly-tiny but nevertheless breaking change queued up over in react-native-firebase and I was looking for something/anything to bundle with it, this fits the bill.